dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -147,7 +148,34 @@ static public void wrap(final VoidDelegate delegate) throws Exception {
             }
         }
     } // wrap.
+    
+    static public void wrapNoException(final VoidDelegate delegate)  {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)